src/vehicle_use.cpp

```diff
@@ -1582,6 +1582,97 @@ void vehicle::use_monster_capture( int part, map */*here*/, const tripoint_bub_m
     invalidate_mass();
 }
 
+void vehicle::use_tiedown_furniture( int part, map *here, const tripoint_bub_ms & )
+{
+    if( !here ) {
+        // should never happen!
+        debugmsg( "vehicle::use_tiedown_furniture() called without a map pointer, please report this error" );
+        return;
+    }
+
+    if( parts[part].is_broken() || parts[part].removed ) {
+        return;
+    }
+
+    const std::string item_var_string = "tied_down_furniture";
+
+    // Why do we construct a copy here?
+    item base_copy = item( parts[part].get_base() );
```

Comment:
Perhaps to not leave things in a broken intermediate state? It doesn't seem necessary here.

Reply:
get_base() returns a const reference, it turns out. Probably best to keep it keep it like that.